### ISSUE TITLE
Unable to crate storage endpoint

### ISSUE THREAD

**Issue by SatishKumbari47** [Azure/azure-cli#11582]
When I tried to create the storage endpoint with the above command(az iot hub routing-endpoint create) using the property values(endpoint-resource-group, --connection-string, endpoint-subscription-id ) from the command (az iot routing-endpoint list), I am unable to create the endpoint and getting error as "exceptionMessage:No valid combination of account information found. If you contact a support representative please include this correlation identifier: 0c78e291-3410-4144-86d4-92aa11e6c237". Please help

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fa3fd83a-be15-aed8-a6b1-a51ff63a91c8
* Version Independent ID: d673ade6-197f-81be-3c55-c9c550c83b50
* Content: [az iot hub routing-endpoint](https://docs.microsoft.com/en-us/cli/azure/iot/hub/routing-endpoint?view=azure-cli-latest#feedback)
* Content Source: [src/azure-cli/azure/cli/command_modules/iot/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/iot/_help.py)
* Service: **iot-hub**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by haroldrandom** [comment]
@zhoxing-ms Please take a look

**Comment by SatishKumbari47** [comment]
@zhoxing-ms Please help me in the process of creating storage endpoint via azure cli. I need to create message routing once after creating the storage endpoint, where I stucked now.

**Comment by SatishKumbari47** [comment]
@haroldrandom  @zhoxing-ms 

Is there someone who can help me on this issue please. That would be grateful.

**Comment by zhoxing-ms** [comment]
Hi, after reading the error information in your feedback, I guess the problem may be related to connection-string. 
Please provide the format and storage settings of the connection-string you are using. Sensitive information can be encrypted with *.

**Comment by SatishKumbari47** [comment]
Hello @zhoxing-ms 

This issue has been resolved, I got the mistake that I was doing while creating endpoint that is I was passing existing already used container name while creating endpoint hence I was facing this issue.
I had no idea that I have to pass a new container name every time for every new endpoint. Anyways, now I could able to create storage endpoint so did message routing. Thanks everyone for your help :)
